### DISCUSSION TITLE
How to get/set tracee policy CRD using kube-api endpoint

### DISCUSSION THREAD

**Discussion by hammadahmed1985** [aquasecurity/tracee#3860] [Adopters]
Hi,
I am trying to get and set the tracee policy using the kube-api. I am running a simple tracee in a simple minikube cluster. I use `kubectl proxy --port=8080` and want to then list the default policy. Then I want to push a new default policy using the API as well. Is that possible?

**Comment by josedonizetti** [comment]
@hammadahmed1985 sure, you mean like? 

```
// get the defaul policy
curl http://localhost:8080/apis/tracee.aquasec.com/v1beta1/policies/default-policy

// delete it
curl -XDELETE http://localhost:8080/apis/tracee.aquasec.com/v1beta1/policies/default-policy

// create a new one
curl -XPOST \
-H 'Content-type: application/yaml'\
--data-binary @path/to/policy.yaml \
http://localhost:8080/apis/tracee.aquasec.com/v1beta1/policies
```

**Reply by hammadahmed1985** [reply]
This is exactly what I was looking for. Thanks!!

**Reply by hammadahmed1985** [reply]
@josedonizetti Do you know how the namespace plays in this? This obviously works but does it mean that these policy have a global scope? My tracee is installed in the `tracee` namespace. Thanks!

**Reply by josedonizetti** [reply]
yeah, currently policies are not namespaced, because the idea is that you should have only one tracee running per cluster. Perhaps in the future we might review to support a namespaced one, but currently they are not.

**Reply by hammadahmed1985** [reply]
Thanks @josedonizetti
